gtda/mapper/cluster.py

```diff
@@ -396,8 +398,14 @@ class FirstSimpleGap(ClusterMixin, BaseEstimator, Agglomerative):
 
     """
 
-    _hyperparameters = {'relative_gap_size': [float, (0, 1)],
-                        'max_fraction': [float, (0, 1)]}
+    _hyperparameters = {
+        'relative_gap_size': {
+            'type': Real, 'in': Interval(0, 1, closed='right')},
+        'max_fraction': {
+            'type': (Real, type(None)), 'in': Interval(0, 1, closed='right')},
```

Comment:
Do we  want to allow `max_fraction=1`?

Reply:
That's the default parameter, in fact.

Reply:
Ok, I confused it with `relative_gap_size`